_inc/client/main.jsx

```diff
@@ -174,36 +182,22 @@ const Main = React.createClass( {
 				pageComponent = <Apps siteRawUrl={ this.props.siteRawUrl } />;
 				break;
 			case '/plans':
-				pageComponent = <Plans siteRawUrl={ this.props.siteRawUrl } siteAdminUrl={ this.props.siteAdminUrl } />;
+				pageComponent = <Plans siteRawUrl={ this.props.siteRawUrl } siteAdminUrl={ this } />;
```

Comment:
This change is not ok, should still have the `props.siteAdminUrl` otherwise the URL for example for "View your spam stats" ends up like `http://example.tld/wp-admin/[object%20Object]admin.php?page=akismet-key-config`